### PR TITLE
YONK-829: Staff Permission Added on Theme Post/Update Views

### PR DESCRIPTION
**Pull request by aamishbaloch**
Changes include Staff Permission Added on Theme Post/Update Views

### CODE REVIEW

**Comment by aamishbaloch** [comment]
@ziafazal Can you review this?

File: mobileapps/tests.py [hunk 1]
```diff
@@ -964,6 +964,32 @@ def test_mobileapps_organization_theme_add(self):
         self.assertIn('image_url_small', response.data['results'][0]['logo_image'])
         self.assertIn('image_url_x-small', response.data['results'][0]['logo_image'])
 
+    def test_mobileapps_organization_theme_add_with_non_staff_user(self):
```
Comment (ziafazal) [inline mobileapps/tests.py:967]: We need similiar test for `OrganizationThemeDetailView ` permissions.

**Review by ziafazal — CHANGES_REQUESTED** [review]
@aamishbaloch we need to add one more test(see inline comments) other than that looks good.

**Comment by aamishbaloch** [comment]
@ziafazal Can you review the updates?

**Review by ziafazal — APPROVED** [review]
👍

**Pull request merged**